src/main/java/com/aquarium/aquarium_backend/databaseTables/EnvironmentalRequirements.java

```diff
@@ -68,14 +65,13 @@ public boolean equals(Object o) {
     EnvironmentalRequirements that = (EnvironmentalRequirements) o;
     return getEnvironmentalRequirementsId() == that.getEnvironmentalRequirementsId()
         && Float.compare(getValue(), that.getValue()) == 0
-        && Objects.equals(getFishType(), that.getFishType())
+        // && Objects.equals(getFishType(), that.getFishType())
         && Objects.equals(getName(), that.getName())
```

Comment:
I need info about changes in the requirements.

Reply:
Same as above. I encountered problem where I couldn't insert fishType because databese wanted to find ID of new record in requirements table